src/packagers/JSPackager.js

```diff
@@ -87,7 +91,15 @@ class JSPackager extends Packager {
       entry.push(this.bundle.entryAsset.id);
     }
 
-    await this.dest.end('},{},' + JSON.stringify(entry) + ')');
+    await this.dest.write('},{},' + JSON.stringify(entry) + ')');
+    // Add sourcemap url
+    await this.dest.write(
```

Comment:
This should only be added if sourcemaps are enabled right?

Reply:
Ow yes, will change this